src/coffee/import.coffee

```diff
@@ -294,18 +303,17 @@ class Import
 
   createOrUpdate: (products, types, matchFn) ->
     Promise.all _.map products, (entry) =>
-      @repeater.execute =>
```

Comment:
Repeater was there to handle HTTP errors which are now handled already in `sphere-node-sdk` module: https://github.com/sphereio/sphere-node-sdk/releases/tag/v1.17.0